.github/CHANGELOG.md

```diff
@@ -6,14 +6,19 @@
 
 ### Improvements
 
+* Change `inv()` method to `qml.adjoint()` method in Python unit tests to be aligned with recent change in Pennylane and avoid CI failures.
```

Comment:
```suggestion
* Update `inv()` to `qml.adjoint()` in Python tests following recent changes in Pennylane.
```

Reply:
Thanks @AmintorDusko ! Let me update it.